pump/storage/storage.go

```diff
@@ -1114,9 +1115,15 @@ func (a *Append) PullCommitBinlog(ctx context.Context, last int64) <-chan []byte
 
 		for {
 			startTS := last + 1
+			limitTS := atomic.LoadInt64(&a.maxCommitTS) + 1
+			if startTS > limitTS {
+				log.Warn("last ts is greater than pump's max commit ts", zap.Int64("last ts", startTS-1), zap.Int64("max commit ts", limitTS-1))
+				time.Sleep(time.Second * 10)
```

Comment:
10 second is too long, how about return directly

Reply:
if so, you also need to check logic of drainer

Reply:
can't return here, otherwise, drainer will not get binlog forever and need restart

Reply:
and this need update a lot of code,  I think just wait is ok

Reply:
10 second is too long

Reply:
update to 1 second